runner/ansible/roles/1.3.7/tasks/main.yml

```diff
@@ -0,0 +1,33 @@
+---
+
+- name: Set Test ID
```

Comment:
This looks outdated too:
Check: https://github.com/trento-project/trento/blob/main/examples/generic-azure.yaml#L496

Reply:
should be OK now